code/modules/clothing/head/helmet.dm

```diff
@@ -17,6 +17,9 @@
 		"Drask" = 'icons/mob/species/drask/helmet.dmi'
 		)
 
+/obj/item/clothing/head/helmet/modern

```

Comment:
@CrAzYPiLoT-SS13 I don't recall the exact details, but I am pretty sure there's a hard limit somewhere over number of different paths you can have